HW_2/task_4.py

```diff
@@ -0,0 +1,27 @@
+"""
+Задайте список из N элементов, заполненный целыми числами из промежутка [-N, N].
+Найдите произведение элементов на индексах, хранящихся в файле indexes.txt (в одной строке один индекс).
+Решение должно работать при любом натуральном N.
+
+Ввод: значение типа <int>
+Вывод: значение типа <int>
+"""
+# Чтобы обращаться к функциям, надо импортировать модуль random 
+# или ипортировать отдельные функции из него: надо написать так
+from random import randint
+num = int(input('Задайте список: '))
+#  Ф randint возвращает случайное целое число в промежутке между двумя значениями min и max
+list = [randint (- num, num) for _ in range(num)]
+print(list)
+# Чтобы работать с файлами, надо завести какую 
+# то переменную, которая будет связана с этим текстовым файлом. 
+res = 1
+with open('indexes.txt', 'r') as file: #Режим r  который позволяет нам читать данные с файла
+# при помощи цикла пробежимся по файлу 
+    for line in file:
+        index = int(line)
+        if len(list) > index >= -len(list): # len(list) - число элементов в списке
```

Comment:
```suggestion
        if num > index >= -num: # len(list) - число элементов в списке
```
Число элементов вас уже известно, считать его ненужно. Особенно дважды..